Platform/AlderlakeBoardPkg/Library/FspmUpdUpdateLib/FspmUpdUpdateLib.c

```diff
@@ -512,19 +512,20 @@ UpdateFspConfig (
     DEBUG((DEBUG_INFO, "ADL_S_ADP_S_CRB board Id %x .....\n", PlatformId));
     CopyMem(SaDisplayConfigTable, (VOID *)(UINTN)mAdlUDdr4RowDisplayDdiConfig, sizeof(mAdlUDdr4RowDisplayDdiConfig));
     break;
+#ifndef PLATFORM_ADLN
   case PLATFORM_ID_ADL_PS_DDR5_RVP:
     DEBUG((DEBUG_INFO, "PLATFORM_ID_ADL_PS_DDR5_RVP board Id %x .....\n", PlatformId));
     CopyMem(SaDisplayConfigTable, (VOID *)(UINTN)mAdlPSEdpHdmiDisplayDdiConfig, sizeof(mAdlPSEdpHdmiDisplayDdiConfig));
     break;
+#else
+  case PLATFORM_ID_ADL_N_DDR5_CRB:
+#endif
```

Comment:
Why adding "#ifndef PLATFORM_ADLN"?

Reply:
because ADL-N boardid is conflict with ADL-PS so putting this conditional check to choose either one of them to avoid the conflict.